tests/assets/ansible/roles/caasp/tasks/Suse.yaml

```diff
@@ -15,6 +15,13 @@
   loop_control:
     loop_var: _repo
 
+- name: Add custom certificates
+  synchronize:
+    src: "{{ playbook_dir }}/../../../localassets/certificates/"
+    dest: /etc/pki/trust/anchors
+  notify:
+    - update-ca-certificates
```

Comment:
Is it possible to disable certifacts at all?

Reply:
You can pass any registries.conf config you want so yes. I would not recommend it though, as we have everything to use them ready and TLS is expected by default by container runtimes.